packages/tools/src/stateManagement/segmentation/helpers/normalizeSegmentationInput.ts

```diff
@@ -13,6 +15,14 @@ function normalizeSegmentationInput(
   segmentationInput: SegmentationPublicInput
 ): Segmentation {
   const { segmentationId, representation } = segmentationInput;
+  const data = { ...representation.data };
+
+  if (representation.type === SegmentationRepresentations.Contour) {
+    // Make sure annotationUIDsMap is defined because an empty contour is
```

Comment:
I noticed that you already have this set as the default, so there's no need to include it in the examples, I suppose.